lib/redis/store/factory.rb

```diff
@@ -20,7 +20,7 @@ def create
         if @addresses.empty?
           @addresses << {}
         end
-        
```

Comment:
make sure you reset this file when you push again, it's a bad practice to include unnecessary changes in your pull request...although I do appreciate the spacing changes, we should probably be doing that across the board as a separate issue.